Node/src/TeamsChatConnector.ts

```diff
@@ -161,6 +166,45 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  *  Return a newly started reply chain address in channel
+  *  @param {string} serverUrl - Server url is composed of baseUrl and cloud name, remember to find your correct cloud name in session or the function will not find the team.
+  *  @param {string} channelId - The channel id, will post in the channel.  
+  *  @param {builder.Message} message - The message to post in the channel.
+  *  @param {function} callback - This callback returns err or result.
+  */
+  public beginReplyChainInChannel(serverUrl: string, channelId: string, message: builder.Message, callback: (err: Error, address: builder.IChatConnectorAddress) => void) : void {
+    var options: msRest.RequestOptions = {customHeaders: {}, jar: false};
+    var restClient = new RestClient(serverUrl, null);
+    var remoteQuery = new RemoteQuery(restClient);
+    this.getAccessToken((err, token) => {
+        if (!err && token) {
+          options.customHeaders = {
+            'Authorization': 'Bearer ' + token
+          };
+          var innerCallback = function (err: Error, result: ReplyResult) {
+            if (result && result.hasOwnProperty("id"))
+            {
+              var address: builder.IChatConnectorAddress = message.toMessage().address;
+              address.channelId = 'msteams';
+              address.conversation.id = result.id;
+              address.id = result.activityId;
```

Comment:
Is activityId always present? If not, check for the presence of activityId before assigning to address.id, or you're create an id property with the value "undefined".